### ISSUE TITLE
32-bit ARM v7

### ISSUE THREAD

**Issue by corasaniti** [anujdatar/duckdns-docker#1]
Hello, do you think you will keep the 32-bit ARM v7 architecture for a long time. 
I noticed that Linuxserver is abandoning the ARMv7 architecture for the 64 Bit one
Thanks
PC

**Comment by anujdatar** [comment]
Hi, I have automated monthly builds for all supported architectures, `armv7`, `armv8` and `amd64`. So as long as Alpine Linux continues to publish an `armv7` image, there should be updated images of this on the first of every month.

While I don't see myself changing that any time soon, there is one caveat. I don't have a proper `armv7` board to test the image on anymore. I will keep testing the image on a Raspberry Pi 4 running `armhf` RPi OS, but I don't know if its the same thing since the Pi4 is actually a `armv8` CPU.

**Comment by corasaniti** [comment]
Thanks

**Comment by anujdatar** [comment]
You're welcome. The ddns update itself is actually just a simple curl request, so I don't see anything breaking there. Just that a couple of years ago there was an issue with alpine linux images for armv7, around version 3.13 and 3.14. So I had to fix the builds to an older version. But that was fixed a while ago, I'll keep an eye out for those kinds of breaking changes.

**Comment by corasaniti** [comment]
@anujdatar 
I'd like to map a log file to a container volume. But I have seen that it does not produce any logs but only prints to standard output. 

I also see that I also set the TZ with my Europe/Rome I see in the log, the time 2 hours in advance (here in Rome it's 13.06)
`[Sun May 28 11:06:18 UTC 2023] DuckDNS update ok`

This is my docker-run
```
docker run -d 
  --name=duckdns\
  --hostname=duckdns \
  --dns=8.8.8.8 \
  -e TZ=Europe/Rome \
  -e SUBDOMAINS=my-domain \
  -e TOKEN=xxxxxxxxxx \
  -e FREQUENCY=5 \
  --restart always \
  anujdatar/duckdns`
```

Do you have any suggestions on how to do it?
Thank you

**Comment by anujdatar** [comment]
Hi, yeah, my setup when I initially started the project was very basic and janky. I never set up logs, just `stdouts`. But that is good to have. I'll start an issue for that. Should take me a couple of days to add I guess, can't work on it right away.

Not sure about the timezone. Seems like the the TZ reset isn't working. I'll look into it.

**Comment by anujdatar** [comment]
@corasaniti oh well, i just took a look at the code. I never added a way to set TZ data, so its just stuck on the default UTC/GMT timezone. I'll have to add that. I created an issue for that too. As of now the TZ environment variable does nothing.

I did not look at linuxserver's containers when i made this, it just started as a small project to learn github actions. I never made it to be directly compatible with others. I'll work on adding this feature as well. Thanks for bringing it up.
